client/src/main/java/io/pravega/client/stream/impl/ControllerImplConfig.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.stream.impl;
+
+import java.io.Serializable;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class ControllerImplConfig implements Serializable {
```

Comment:
Are we debating making this class public or allowing clients to overriding the retry config? If its just about making this class public, I had to do this because the clients creating this instance are in a different package. If we don't want clients to override the default retry config then as commented above I can remove this config entirely, please let me know.